server/src/main/java/org/elasticsearch/script/field/EmptyField.java

```diff
@@ -15,6 +15,9 @@
  * Script field with no mapping, always returns {@code defaultValue}.
  */
 public class EmptyField<T> extends Field<T> {
+
+    /* ---- Empty Field Members ---- */
```

Comment:
What's this mean?

Reply:
Ah, I added this for consistency with the other fields where I was marking sections with the actual field members, converter helper methods, and converter. Happy to remove this and any others if you want. For the others I find it easier to scan the file with these comments as markers and it creates a consistency in ordering through each field class.

Reply:
Ya remove the comment where it doesn't make sense.

Reply:
Removed.